Wrap armies with more than 2 units

gereon77 opened the pull request:
Closes #19 

Preview:

![image](https://user-images.githubusercontent.com/22304202/81090424-febeab00-8efd-11ea-9472-6b47ecd25f56.png)
![image](https://user-images.githubusercontent.com/22304202/81090598-4a715480-8efe-11ea-9307-bbceac397531.png)

Longwelwind commented:
Few things:

* It looks slick!
* Why making a `noWrap` attribute and not just making long enough boxes so that, for these regions, units will never wrap?

gereon77 commented:
>     * Why making a `noWrap` attribute and not just making long enough boxes so that, for these regions, units will never wrap?

I thought about that and in fact I changed all unitSlots in Tiled to be 65x65 except Sunset and all ports where i changed the Size to 100x32. I did it to have a feeling in Tiled where units will be placed. You are right I could omit the `noWrap` property and always apply the width.

There is one additional thing which came to my mind. In sea areas it might be nicer if we would apply `flex-wrap: wrap` instead of `flex-wrap: wrap-reverse`. I guess 3 ships look nicer with 2 top ships and 1 botton ships. What to do you think?

I fix it and send a screenshot with all areas containing 3 units.

gereon77 commented:
Ok it was absolutely necessary to test all regions. `maxWidth` is causing issues in the east when the slot goes outside the map then:

![image](https://user-images.githubusercontent.com/22304202/81183357-5c5f0000-8faf-11ea-9855-1f6699d7d98e.png)

We need to use `width` and now it looks fine:

![image](https://user-images.githubusercontent.com/22304202/81182781-9085f100-8fae-11ea-8789-56aa46078ecd.png)
![image](https://user-images.githubusercontent.com/22304202/81183247-35083300-8faf-11ea-88be-476f988b933c.png)

I tried to make as much info (Castles, Barrels, Stronghold, Name) visible as possible. That is the best compromise I found and that's why Dragonstone is the only land area which wraps after 3 units. I did this as the stronghold, barrel, crown and the garrison is so near together in dragonstone that this was the best solution. With a 4 unit army (and only then) the barrel and crown is hidden though.
